README.md

```diff
@@ -188,6 +199,14 @@ To run the tests behind proxy using Notebook:
    * kfp_v2
    * training (except TFJob due to https://github.com/canonical/training-operator/issues/182)
 
+#### Running using `driver`
+
+You can pass the `--proxy` flag and set the values for proxies to the tox command and this should automatically apply the required changes to run behind proxy.
+
+```bash
+tox -e kubeflow-<local|remote> -- --proxy http_proxy="http_proxy:port" https_proxy="https_proxy:port" no_proxy="<cluster cidr>,<service cluster ip range>,127.0.0.1,<nodes internal ip(s)>/24,<cluster hostname>,.svc,.local"
```

Comment:
this might leave room for error if users don't set the key correctly, for example `--proxy http-proxy=..` will still run, right? we could:
1. Add a check in `proxy_context` method that the keys are correct, and log a helpful error if they are incorrect
2. Have a separate argument for every key of `http_proxy` `https_proxy` `no_proxy`

Reply:
I have added a check, now it always expects three args and fails when it receives less or none.